common/config/rush/common-versions.json

```diff
@@ -98,6 +98,7 @@
 
     // Use two different versions of @types/webpack-dev-server to allow pnpmfile.js to bring in
     // two different versions of the webpack typings
-    "@types/webpack-dev-server": ["3.11.2"]
+    "@types/webpack-dev-server": ["3.11.2"],
+    "eslint": ["8.7.0"]
```

Comment:
Why an alternate version?

Reply:
eslint is also used in other packages. I was changing it for a single package.